app/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+from flask import Flask
+
+app = Flask(__name__)
+app.config.from_object('config')
+
+from app import views
```

Comment:
Line break at the end of file.

At least two things you need to set in your editor:
* Ensure newline at end-of-file.
* Don't use tabs for indenting.  Python files should have 4 spaces per indentation level. HTML/CSS/JS can be 2 or 4 (We use 2 spaces for frontend).